spartan/scripts/upgrade_rollup_with_cli.sh

```diff
@@ -135,19 +135,4 @@ else
   EXE="node --no-warnings $AZTEC_BIN"
 fi
 
-
-# if DEPOSIT_AMOUNT is set, we deposit governance tokens
-if [ -n "$DEPOSIT_AMOUNT" ]; then
-  echo "Depositing $DEPOSIT_AMOUNT governance tokens to $MY_ADDR"
-  $EXE deposit-governance-tokens -r $REGISTRY --recipient $MY_ADDR -a $DEPOSIT_AMOUNT $MINT
-fi
-
-OUTPUT=$($EXE deploy-new-rollup -r $REGISTRY --salt $SALT --json $TEST_ACCOUNTS $SPONSORED_FPC)
-
-PAYLOAD=$(echo $OUTPUT | jq -r '.payloadAddress')
-
-PROPOSAL_ID=$($EXE propose-with-lock -r $REGISTRY --payload-address $PAYLOAD --json | jq -r '.proposalId')
-
-$EXE vote-on-governance-proposal --proposal-id $PROPOSAL_ID --in-favor true --wait true -r $REGISTRY
-
-$EXE execute-governance-proposal --proposal-id $PROPOSAL_ID --wait true -r $REGISTRY
+$EXE deploy-new-rollup -r $REGISTRY --salt $SALT --json $TEST_ACCOUNTS $SPONSORED_FPC
```

Comment:
We dont want to stop testing this functionality, even if we are not using it for the existing deployment

Reply:
The things that we stop testing inside the test using this:
- minting and depositing funds into gov
- proposing with lock
- voting on our own proposal
- executing it

All things that are already tested in the solidity contracts. 
The thing we kept in is essentially the new deployment making it into registry, and seeing that they when restarted will switch to and use that. 

So while we are not doing it from the aztec-cli, I would say this functionality is still tested, just in the solidity.

Reply:
Am adding an explicit solidity tests for this so it is simpler to see 👍